pkg/features/kube_features.go

```diff
@@ -1052,9 +1052,9 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	TopologyManagerPolicyAlphaOptions: {Default: false, PreRelease: featuregate.Alpha},
 
-	TopologyManagerPolicyBetaOptions: {Default: false, PreRelease: featuregate.Beta},
+	TopologyManagerPolicyBetaOptions: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
AFAIU this is a "bug fix" meaning that this should have defaulted to `true` since the beginning. If so I'd isolate the change in its own commit

Reply:
Yeah, good idea :)

Reply:
This change was moved to separate commit.

Reply:
thanks, this part LGTM